Make.inc

```diff
@@ -1418,7 +1418,7 @@ WINE ?= wine
 # many of the following targets must be = not := because the expansion of the makefile functions (and $1) shouldn't happen until later
 ifeq ($(BUILD_OS), WINNT) # MSYS
 spawn = $(1)
-cygpath_w = $(1)
+cygpath_w = `cygpath -w $(1)`
```

Comment:
This seems suspicious to me; is `cygpath` always available in an MSYS environment?

Reply:
Yes

> MSYS2 ships the Cygwin tool cygpath by default which allows converting paths between the Unix format, Windows format, and mixed format
> https://www.msys2.org/docs/filesystem-paths/#manual-unix-windows-path-conversion